data_architect_misc/data_transformer/transform_functions/common_transform_functions.py

```diff
@@ -16,6 +16,7 @@
 import numpy as np
 
 import transform_errors
+from constants import comp_harm_constants
```

Comment:
We don't need to import comp_harm_constants into common_transform_functions.py. This file is NOT related to comp harm and should only keep the functions that are NOT related to comp harm.